ircd/s_conf.c

```diff
@@ -804,9 +804,9 @@ set_default_conf(void)
 	ConfigChannel.channel_target_change = true;
 	ConfigChannel.disable_local_channels = false;
 	ConfigChannel.displayed_usercount = 3;
+	ConfigChannel.opmod_send_statusmsg = false;
 
 	ConfigChannel.autochanmodes = MODE_TOPICLIMIT | MODE_NOPRIVMSGS;
```

Comment:
Please move this line up to restore the gap between `ConfigChannel` and `ConfigServerHide`